packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -92,6 +92,9 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
 
   const onTexFieldKeyDown = useCallback(
     (ev: React.KeyboardEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+      if (ev.nativeEvent.isComposing) {
```

Comment:
Please add a comment in the code here explaining why this is needed. It's very easy to forget the reason for such checks.